sql/core/src/test/scala/org/apache/spark/sql/sources/SaveLoadSuite.scala

```diff
@@ -142,4 +143,15 @@ class SaveLoadSuite extends DataSourceTest with SharedSQLContext with BeforeAndA
       assert(e.contains(s"Partition column `$unknown` not found in schema $schemaCatalog"))
     }
   }
+
+  test("skip empty files in load") {
+    withTempDir { dir =>
+      val path = dir.getCanonicalPath
+      Files.write(Paths.get(path, "empty"), Array.empty[Byte])
+      Files.write(Paths.get(path, "notEmpty"), "a".getBytes)
+      val readback = spark.read.option("wholetext", true).text(path)
+
+      assert(readback.rdd.getNumPartitions == 1)
```

Comment:
Do we need `1 === ...` to get the right assert message? it's tiny.

Reply:
It seems expected value should be on right. I changed the order and got the following:
```scala
assert(123 === readback.rdd.getNumPartitions)
```
```
123 did not equal 1
ScalaTestFailureLocation: org.apache.spark.sql.sources.SaveLoadSuite at (SaveLoadSuite.scala:155)
Expected :1
Actual   :123
```

Current assert triggers correct message:
```scala
assert(readback.rdd.getNumPartitions == 123)
```
```
1 did not equal 123
ScalaTestFailureLocation: org.apache.spark.sql.sources.SaveLoadSuite at (SaveLoadSuite.scala:155)
Expected :123
Actual   :1
```

Reply:
I am just referring to === and the order of args. I'm sure the test was right as-is in what it asserts.